loops-and-arrays.js

```diff
@@ -5,6 +5,9 @@
 // ==========================================
 
 const names = ["Henk", "Piet", "Fred", "Joop"];
+for (let i = 0; i <= names.length-1; i++){
```

Comment:
De output klopt, dus misschien dat jouw code gewoon helemaal goed is, maar ik snap niet goed waarom je gebruik maakt van het = teken. Volgens mij heb je daarom gebruik gemaakt van de -1, maar is dat niet nodig als je ze beiden weglaat. Hij moet 'm vier keer loopen, en dat is de names.length.